validator/sawtooth_validator/database/indexed_database.py

```diff
@@ -117,7 +117,10 @@ def contains_key(self, key, index=None):
             search_db = self._main_db
 
         with self._lmdb.begin(db=search_db) as txn:
-            return txn.cursor().set_key(key.encode())
+            try:
+                return txn.cursor().set_key(key.encode())
+            except lmdb.BadValsizeError:
```

Comment:
This should probably log something.

Reply:
What are you thinking? Something like _"Checked for existence of empty key"_?